deployment/terraform/cluster/local/apps/variables.tf

```diff
@@ -264,7 +325,7 @@ variable "ad-manager" {
 variable "modelservice" {
   type = "map"
   default = {
-    enabled = false
+    enabled = true
     modelservice_instances = 1
```

Comment:
Intentional? My assumption is that Haystack wants the Haystack ones true and the AA ones false for now. Is that correct?